sdk/search/Azure.Search.Documents/apis.txt

```diff
@@ -69,10 +69,10 @@ namespace Azure.Search.Documents
         Task<Response<DataSource>> GetDataSourceAsync(string, SearchRequestOptions, CancellationToken) { }
         Response<IReadOnlyList<DataSource>> GetDataSources(IEnumerable<string>, SearchRequestOptions, CancellationToken) { }
         Task<Response<IReadOnlyList<DataSource>>> GetDataSourcesAsync(IEnumerable<string>, SearchRequestOptions, CancellationToken) { }
+    }
 
-
-
-
+    public class SearchIndexClient
```

Comment:
Is it better to have an API to go back `SearchServiceClient`?

Reply:
Our guidance in the past has always been "have accessors move down the hierarchy."  It might be worth reconsidering that here though if we're giving the best `SearchClient` name to the document operations.